### ISSUE TITLE
Reset the arcore the session in android secneform

### ISSUE THREAD

**Issue by jayaprakash8088** [google-ar/arcore-android-sdk#1117]
### SPECIFIC ISSUE ENCOUNTERED

Not able to reset or restart the session
### VERSIONS USED
- Android Studio: 4.0.1
- ARCore SDK for Android: sceneform
- Device manufacturer, model, and O/S: android
- ARCore: 
  

 In Ar kit , there is functionality for restart or reset the session.But in ARcore sceneform we don`t have function to reset the session like
*session.close()
*session.resume()
*session.pause().
We can restart the session only after refreshing the Ar core sceneform, but i need to restart without refreshing the Arcore.
Is there any way to reset or restart a session in ArCore ? Or any alternative approach to achieve this ?

**Comment by newmluser** [comment]
what version of sceneform are you using

**Comment by devbridie** [comment]
Thanks for the report. I'm trying to understand why the results of the session wouldneed to be reset-- ARCore tries to continually update its understanding of the world, and that's probably why we don't support resetting a session. Could you explain the usecase for this? Is restarting the session with a new activity or fragment not an acceptable workaround?

**Comment by martin96depe** [comment]
Hi, I have the same problem. Specifically I am trying to address several problems that came with repetitive patterns in the environment. The most annoying  issue is the drift that occurs when Arcore finds similar regions in the scene and tries to correct the virtual objects in the scene. What I have tried so far is detecting these drifts and when they occur I detach all anchors and create a new anchor in the current camera position where to associate the reloaded renderables later on. 

I Have done many trials and I found that the process of detecting drifts, detaching anchors, create new anchor and reload model become instable the more drifts are detected. For this reason it will be great if there would be a method like Session.clear() or Session.reset() that simply reset the session so that the user has to rescan the scene looking for a plane.

IMO resetting the scene/session is the best option when it comes to deal with repetitive patterns since I think these introduce a lot of noise in the Arcore's environment understanding
